front50-core/src/test/groovy/com/netflix/spinnaker/front50/model/application/ApplicationModelSpec.groovy

```diff
@@ -47,4 +50,18 @@ class ApplicationModelSpec extends Specification {
     listApp.cloudProviders == 'a,b'
     normalApp.cloudProviders == 'a,b,c'
   }
+
+  void 'ignore correct timestamped json properties from application object to json'() {
+
+    given:
+    def application = new Application()
+    application.setCreatedAt(new Long(1))
+    application.setLastModifiedBy("foo")
+
+    ObjectMapper mapper = new ObjectMapper().addMixIn(Timestamped.class, TimestampedMixins.class)
+    String appJSON = mapper.writeValueAsString(application)
```

Comment:
can use `mapper.writerWithDefaultPrettyPrinter().writeValueAsString(application)` to make your assertion below more readable.

Reply:
somehow I made it uglier ;(